aiidalab/app.py

```diff
@@ -752,12 +839,18 @@ def _refresh_versions(self):
                 if self.include_prereleases or not parse(version).is_prerelease
             ]
 
-    @throttled(calls_per_second=1)
+    def _refresh_dependencies_to_install(self):
+        self.dependencies_to_install = self._app.find_dependencies_to_install(
+            self.version_to_install
+        )
+
+    @throttled(interval=10)
```

Comment:
This change is important. Otherwise every 1s the app will refresh which is too short. I don't know why this feature is not working. But the blink of the app in every 1s seems like a bug but actually the expected behavior.